kalliope/core/ConfigurationManager/SettingLoader.py

```diff
@@ -440,13 +440,18 @@ def _get_rest_api(settings):
                 if not 1024 <= port <= 65535:
                     raise SettingInvalidException("port must be in range 1024-65535")
 
+                # check the CORS request settings
+                allowed_cors_origin = rest_api["allowed_cors_origin"]
```

Comment:
if "allowed_cors_origin" is not in setting this line will fail. You should replace with something like : 
```
allowed_cors_origin = False
if "allowed_cors_origin" in rest_api:
     allowed_cors_origin = rest_api["allowed_cors_origin"]
```